### PR TITLE
[next.config.js]: More securitah.

### PR DESCRIPTION
**Pull request by dilyevsky**
Add https://github.com/jagaapple/next-secure-headers/tree/master#rules which sets `X-Frame-Options` among other things.

### CODE REVIEW

**Review by dylanjha — APPROVED** [review]

File: next.config.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const { createSecureHeaders } = require("next-secure-headers");
```
Comment (dylanjha) [inline next.config.js:1]:
```
23:13:10.024 | $ /vercel/workpath0/node_modules/.bin/eslint . --ext .js,.jsx,.ts,.tsx
-- | --
23:13:11.973 | Warning: React version not specified in eslint-plugin-react settings. See https://github.com/yannickcr/eslint-plugin-react#configuration .
23:13:13.083 | /vercel/workpath0/components/record-page.tsx
23:13:13.083 | 346:20  warning  Missing semicolon  semi
23:13:13.083 | /vercel/workpath0/next.config.__vercel_builder_backup__.js
23:13:13.084 | 1:33  error  Require statement not part of import statement  @typescript-eslint/no-var-requires
23:13:13.084 | ✖ 2 problems (1 error, 1 warning)
23:13:13.084 | 0 errors and 1 warning potentially fixable with the `--fix` option.
23:13:13.131 | error Command failed with exit code 1.

```

Vercel linter doesn't like this require statement, try

```
import createSecureHeaders from 'next-secure-headers'
```

Comment (dilyevsky) [reply next.config.js]: Fixed - still failing

Comment (dylanjha) [reply next.config.js]: New failure, awesome

```
23:37:18.013 | /vercel/workpath0/next.config.__vercel_builder_backup__.js:1
-- | --
23:37:18.013 | import createSecureHeaders from 'next-secure-headers';
23:37:18.013 | ^^^^^^
23:37:18.013 | SyntaxError: Cannot use import statement outside a module
23:37:18.013 | at wrapSafe (internal/modules/cjs/loader.js:1053:16)

js
```

Let me try something else

Comment (dylanjha) [reply next.config.js:1]: Looks like next.js injects this file `/vercel/workpath0/next.config.__vercel_builder_backup__.js`

our eslint rules did not like it. Now that I think about it, we might actually want eslint to ignore the whole `./vercel` directory

**Review by mmcc — APPROVED** [review]

**Comment by dylanjha** [comment]
thanks @dilyevsky !

**Pull request merged**